packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -57,11 +70,21 @@ describe( 'General media replace flow', () => {
 				name: 'Replace',
 			} )
 		);
-
 		const uploadMenu = screen.getByRole( 'menu' );
 
 		expect( uploadMenu ).toBeInTheDocument();
 		expect( uploadMenu ).not.toBeVisible();
+
+		/**
+		 * The popover will be displayed and positioned with a slight delay,
+		 * so we're opting to wait until that happens.
+		 */
+		await waitFor( () =>
+			expect( getWrappingPopoverElement( uploadMenu ) ).toHaveStyle( {
+				top: '13px',
+				left: '0',
```

Comment:
The styles are unfortunately overspecified, we'd rather be checking for "`top` and `left` styles are not empty", but `.toHaveStyle` only supports checking for exact values.

All the tests will need to be updated whenever the positioning algorithm changes, although they never care about what the position exactly is.

Reply:
I agree with you and have also been pestered by the specificity here. So I went for alternatively checking that we have any top and left in e59733d617e41042e51966bde87fed317e1fb525. LMK what you think.